BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -234,48 +225,66 @@ public final class ConvivaAnalytics: NSObject {
      */
     public func pauseTracking() {
         // AdStart is the preferred way to pause tracking according to conviva.
-        client.adStart(sessionKey,
-                       adStream: .ADSTREAM_SEPARATE,
-                       adPlayer: .ADPLAYER_SEPARATE,
-                       adPosition: .ADPOSITION_PREROLL)
-        client.detachPlayer(sessionKey)
+        videoAnalytics?.reportAdBreakStarted(AdPlayer.ADPLAYER_SEPARATE,
+                                             adType: AdTechnology.CLIENT_SIDE,
+                                             adBreakInfo: [:])
```

Comment:
Nit: (Not sure which code-style is used here nowadays 🙂) **all multiline method calls**
```suggestion
        videoAnalytics?.reportAdBreakStarted(
            AdPlayer.ADPLAYER_SEPARATE,
            adType: AdTechnology.CLIENT_SIDE,
            adBreakInfo: [:]
        )
```

Reply:
It looks like either is acceptable to SwiftLint. The current code is what I see more often than the provided code.

Reply:
Both styles are fine for me then 🙂